librocco_receipt_printer/print_queue.py

```diff
@@ -27,10 +27,34 @@ def stream_changes(self, since="now", selector=None):
                 yield change
             time.sleep(1)
 
+    def process_job(self, job):
+        if job.status != "PENDING" or job.printer_id != self.printer_id:
+            return
+
+        print(f"Received job {job.id} for printer {job.printer_id}")
+        job.processing(self.db)
+
+        print(f"Printing job {job.id}...")
+        # We're sleeping for 2 secs to simulate the printing process
+        # TODO: Remove this when we connect to the printer.
+        time.sleep(2)
```

Comment:
This `time.sleep` call would be a better fit in the `print` method of the `DevPrinter` class.